### ISSUE TITLE
[Bug]: sp-top-nav should use a role=navigation and recommend labelling

### ISSUE THREAD

**Issue by jnurthen** [adobe/spectrum-web-components#3401]
### Code of conduct

- [X] I agree to follow this project's code of conduct.

### Impacted component(s)

sp-top-nav

### Expected behavior

A top nav should use be a navigation landmark and should allow the user to label it.

### Actual behavior

Currently it has no landmark.

### How can we reproduce this issue?

1. Go to https://opensource.adobe.com/spectrum-web-components/components/top-nav/
2. Note that top nav doesn't expose any landmarks

**Comment by jnurthen** [comment]
While adding role="navigation" aria-label="Page" to the sp-top-nav element works I don't think a developer should have to add the role themselves. I can't see any instances when role=navigation would not be appropriate for this component.

aria-label is more tricky and we should perhaps add an attribute to allow the developer to specify the accName for the navigation region without requiring one (a single navigation landmark on a page doesn't need a name - it is only needed if there is more than one)
